src/tests/suite/dfs_unit_test.c

```diff
@@ -104,7 +105,7 @@ dfs_test_mount(void **state)
 	assert_int_equal(rc, 0);
 
 	/** destroy the containers */
-	rc = daos_cont_destroy(arg->pool.poh, cuuid, 0, NULL);
+	rc = daos_cont_destroy(arg->pool.poh, str, 0, NULL);
```

Comment:
Suggest pass "cont1" as the argument to daos_cont_destroy() here. i.e., in that case there would be no need to unparse cuuid into str.

Reply:
Just fixed. Will push to git soon.